### ISSUE TITLE
It's throw error in some params when create MergeDisucussion

### ISSUE THREAD

**Issue by shoyuf** [jdalrymple/gitbeaker#3243]
**Description**

Node.js version: 20
Gitbeaker version: 38.0.1
Gitbeaker release (cli, node, browser, core, requester-utils): rest
OS & version: macOS

**Steps to reproduce**

https://github.com/jdalrymple/gitbeaker/blob/fcf745552dee6c0134a0a615a63d73ff92209ca3/packages/core/src/templates/ResourceDiscussions.ts#L117

when I use these codes will throw error `"error": "position[line_range] is invalid"`
```js
const init = () => {
  api.MergeRequestDiscussions.create(ProjectId, MergeRequestIid, "body", {
    commitId: "f7f7471d28571a5947e46d68400f294d881a45d2",
    position: {
      base_sha: "f5ffa7ba5071694e9170f721b72b7da1a5c7c2df",
      start_sha: "f5ffa7ba5071694e9170f721b72b7da1a5c7c2df",
      head_sha: "f7f7471d28571a5947e46d68400f294d881a45d2",
      position_type: "text",
      new_line: "2",
      new_path: "index.js",
      line_range: {
        start: {
          type: "new",
          line_code: "1",
        },
        end: {
          type: "new",
          line_code: "2",
        },
      },
    },
  })
    .then(console.log)
    .catch(console.log);
};
init();
```
**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by shoyuf** [comment]
I can only use a strange method to avoid this problem. I will replace the line_range property with these codes.

```js
"line_range][start][type": "new",
"line_range][start][line_code": 1,
"line_range][end][type": "new",
"line_range][end][line_code": 2,
```

**Comment by jdalrymple** [comment]
Thats my bad, i didnt account for nested props.

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`38.3.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/38.3.0) :rocket:
